src/controllers/user.js

```diff
@@ -34,7 +34,26 @@ const getSingleUser = async (req, res) => {
   }
 };
 
+const deleteProfile = async (req, res) => {
```

Comment:
i think we should also add a feature like deleting the products from the product database which providerId:req.user.userid and isTransactionCompleted:false